go.mod

```diff
@@ -14,9 +14,9 @@ require (
 	github.com/onsi/gomega v1.10.3
 	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/operator-framework/api v0.5.1
+	github.com/operator-framework/api v0.5.0
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-00010101000000-000000000000
-	github.com/operator-framework/operator-registry v1.13.6
+	github.com/operator-framework/operator-registry v0.0.0-00010101000000-000000000000
```

Comment:
This looks off me to me. Any reason we're not using the correct semantic tag?

Reply:
This is for the replaces section, so we'd know we're pointing to the staged operator-registry repo instead of a vendored version.

Reply:
Gotcha, yeah that makes more sense the more I think about it.

Reply:
Any reason we wouldn't be able to use `v0.0.0` instead? I'm not particularly worried, either way, but I think that's a bit clearer to me that's using a local package reference vs. the pseudo-semantic tag that Go spits out when it can't find a proper semantic tag.